src/library_syscall.js

```diff
@@ -227,7 +227,7 @@ var SyscallsLibrary = {
   },
   __syscall5: function(which, varargs) { // open
     var pathname = SYSCALLS.getStr(), flags = SYSCALLS.get(), mode = SYSCALLS.get() // optional TODO
-    var stream = FS.open(pathname, flags, mode);
+    var stream = FS.open(pathname, flags | {{{ cDefine('O_LARGEFILE') }}}, mode);
```

Comment:
Because https://github.com/kripken/emscripten/pull/4813/files#diff-cffbb10ecdf95ae5cfcc9a524a924a75R246 : `sys_open` calls changed to automatically `| O_LARGEFILE`, and calls to `sys_open` that explicitly set that flag now do so implicitly (https://github.com/kripken/emscripten/pull/4813/files/b34ee72dd065960805c911ea10bd4d1ce30c8b09#diff-3042107f3f2299532432901f2c0200ffL13)

This seemed like the right place to do this because we previously forwarded `syscall_cp` as just `syscall` (https://github.com/kripken/emscripten/blob/incoming/system/lib/libc/musl/src/internal/syscall.h#L60), so it seemed sane to forward `sys_open` directly to `syscall` as well.
The better place to do this is probably in the `syscall.h` file itself, rewriting `__sys_open2` and `__sys_open3`.